EndlessClient/Dialogs/GuildDialog.cs

```diff
@@ -136,7 +150,100 @@ private void SetState(GuildDialogState state)
                         AddItemToList(bankAccountItem, sortList: false);
                     }
                     break;
+
+                case GuildDialogState.Management:
+                    {
+                        ListItemType = ListDialogItem.ListItemStyle.Large;
+                        Buttons = ScrollingListDialogButtons.BackCancel;
+
+                        BackAction += (_, _) => SetState(GuildDialogState.Initial);
+
+                        var modifyGuildItem = new ListDialogItem(this, ListDialogItem.ListItemStyle.Large, 0)
+                        {
+                            ShowIconBackGround = false,
+                            IconGraphic = _dialogIconService.IconSheet,
+                            IconGraphicSource = _dialogIconService.GetDialogIconSource(DialogIcon.GuildModify),
+                            PrimaryText = _localizedStringFinder.GetString(EOResourceID.GUILD_MODIFY_GUILD),
+                            SubText = _localizedStringFinder.GetString(EOResourceID.GUILD_CHANGE_YOUR_GUILD_DETAILS),
+                            OffsetY = 45,
+                        };
+                        modifyGuildItem.LeftClick += (_, _) => CheckAndChangeState(GuildDialogState.Modify);
+                        modifyGuildItem.RightClick += (_, _) => CheckAndChangeState(GuildDialogState.Modify);
+
+                        AddItemToList(modifyGuildItem, sortList: false);
+
+                        var manageRankingItem = new ListDialogItem(this, ListDialogItem.ListItemStyle.Large, 0)
+                        {
+                            ShowIconBackGround = false,
+                            IconGraphic = _dialogIconService.IconSheet,
+                            IconGraphicSource = _dialogIconService.GetDialogIconSource(DialogIcon.GuildRanking),
+                            PrimaryText = _localizedStringFinder.GetString(EOResourceID.GUILD_RANKING),
+                            SubText = _localizedStringFinder.GetString(EOResourceID.GUILD_MANAGE_MEMBER_RANKINGS),
+                            OffsetY = 45,
+                        };
+                        manageRankingItem.LeftClick += (_, _) => CheckAndChangeState(GuildDialogState.ManageRankings);
+                        manageRankingItem.RightClick += (_, _) => CheckAndChangeState(GuildDialogState.ManageRankings);
+
+                        AddItemToList(manageRankingItem, sortList: false);
+
+                        var assignRankItem = new ListDialogItem(this, ListDialogItem.ListItemStyle.Large, 0)
+                        {
+                            ShowIconBackGround = false,
+                            IconGraphic = _dialogIconService.IconSheet,
+                            IconGraphicSource = _dialogIconService.GetDialogIconSource(DialogIcon.GuildRanking),
+                            PrimaryText = _localizedStringFinder.GetString(EOResourceID.GUILD_RANKING),
```

Comment:
Is this supposed to be the same text as the above item? (see line 180)

Remember I don't have friends so I'm totally unfamiliar with how the guild dialog is supposed to work 🤣

Reply:
Yeah it does use the same "Ranking" text :P